package.json

```diff
@@ -2,17 +2,38 @@
   "name": "accounts-cbd-int",
   "description": "accounts.cbd.int",
   "dependencies": {
-    "bower": "1.6.5",
-    "colors": "1.1.2",
-    "ejs": "^2.5.7",
-    "express": "4.13.4",
-    "http-proxy": "1.12.0",
-    "morgan": "1.6.1"
+    "@bower_components/angular": "angular/bower-angular#1.4.7",
+    "@bower_components/angular-animate": "angular/bower-angular-animate#1.4.7",
+    "@bower_components/angular-cookies": "angular/bower-angular-cookies#1.4.7",
+    "@bower_components/angular-flex": "git://github.com/scbd/angular-flex.git#1.0.3",
+    "@bower_components/angular-route": "angular/bower-angular-route#1.4.7",
+    "@bower_components/angular-sanitize": "angular/bower-angular-sanitize#1.4.7",
+    "@bower_components/bootstrap": "twbs/bootstrap#3.3.5",
+    "@bower_components/bootstrap-duallistbox": "istvan-ujjmeszaros/bootstrap-duallistbox#2.1.1",
+    "@bower_components/font-awesome": "FortAwesome/Font-Awesome#4.3.0",
+    "@bower_components/jquery": "jquery/jquery-dist#2.1.4",
+    "@bower_components/lodash": "lodash/lodash#3.9.3",
+    "@bower_components/outdated-browser": "burocratik/outdated-browser#1.1.2",
+    "@bower_components/requirejs": "jrburke/requirejs-bower#2.1.22",
+    "@bower_components/text": "requirejs/text#2.0.14",
+    "@bower_components/urijs": "medialize/URI.js#1.16.1",
+    "ejs": "2.5.8",
+    "express": "4.16.3",
+    "http-proxy": "1.17"
   },
-  "engine": "node >= 4.2",
+  "engine": "node >= 8",
   "private": true,
   "scripts": {
-    "postinstall": "./node_modules/.bin/bower install --allow-root",
     "start": "node server"
+  },
+  "engines": {
+    "yarn": ">= 1.6.0"
+  },
+  "resolutions": {
```

Comment:
can we remove this?

Reply:
@blaisefonseca can we?  --flat must be removed if we remove resolutions